src/test/resources/org/rust/lang/core/parser/fixtures/partial/types.rs

```diff
@@ -1 +1,3 @@
 fn foo<'a, 'b>() where for<'c> 'a : 'b {}
+
+type T = () + A;
```

Comment:
Let's add something like `type T = FnOnce(&mut i32) -> () + Send` into complete parser tests to ensure that it's parsed correctly